benchmarks/vector_db_compare/test_minima_qdrant_runner.py

```
@@ -0,0 +1,677 @@
+            hook.write_text("#!/usr/bin/env bash\nprintf '2\\n'\n", encoding="utf-8")
+            hook.chmod(0o755)
+            environment = dict(os.environ)
+            environment.pop("QDRANT_STORAGE_PATH", None)
+            environment.update({
+                "QDRANT_URL": "http://127.0.0.1:1",
+                "QDRANT_RESTART_HOOK": str(hook),
+                "QDRANT_SERVER_PID": "1",
+                "RUN_DIR": str(Path(directory) / "run"),
+            })
+            script = Path(__file__).resolve().parents[2] / "scripts" / "bench_minima_qdrant.sh"
+            result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)